internal/docker/env.go

```diff
@@ -284,6 +285,8 @@ func isOldMinikube(ctx context.Context, minikubeClient k8s.MinikubeClient) bool
 	return minMinikubeVersionBuildkit.GTE(vParsed)
 }
 
+var ddLinux *regexp.Regexp = regexp.MustCompile(`^unix:///home/[^/]+/.docker/desktop/docker.sock$`)
```

Comment:
can we be a bit more permissive here? e.g., `unix:///.*/.docker/desktop/docker.sock`?

i have definitely worked on unix systems where the home directories did not match this regexp, and the relevant manuals note that we shouldn't be assuming they're laid out this way (cf https://refspecs.linuxfoundation.org/FHS_3.0/fhs/ch03s08.html)

Reply:
👍